src/Symfony/Component/Config/Resource/ResourceInterface.php

```diff
@@ -26,8 +26,6 @@ interface ResourceInterface
      * to be identical for different ResourceInterface instances referring to the same
      * resource; and it should be unlikely to collide with that of other, unrelated
      * resource instances.
-     *
-     * @return string A string representation unique to the underlying Resource
      */
-    public function __toString();
+    public function __toString(): string;
```

Comment:
Couldn't we remove that method completely by declaring…
```php
interface ResourceInterface extends Stringable
```

Reply:
We prefer keeping the comment, don't we?

Reply:
Fair enough.